app/Nova/Resources/Organization.php

```diff
@@ -50,15 +51,6 @@ public function fields(NovaRequest $request): array
                 ->sortable()
                 ->rules('required', 'max:255'),
 
-            Text::make('Invoice Email', 'invoice_email')
```

Comment:
Everything looks fine, but now you can create an `Organization` with only an `invoiceEmail` & no `VAT` or vice versa. This might lead to unintended results.